config/routes.rb

```diff
@@ -7,5 +7,5 @@
 
   # Defines the root path route ("/")
   # root "posts#index"
-  resources :policies, param: :policy_id, only: %i[show create]
+  resources :policies, param: :policy_id, only: %i[show create index]
```

Comment:
```suggestion
  resources :policies, only: %i[show create index]
```

será que precisa citar o param aqui se ele for o Id que for a chave primária do banco?

Reply:
vou testar

Reply:
@glmsilva Não funcionou, pode ser que tenha alguma configuração para funcionar mas desconheço.

Reply:
Imagino que o nome da chave por padrão seja apenas `id` e não `policy_id`, caso queira que fique explícito como policy_id aí sim é necessário passar esse `param`, mas se remover igual o @glmsilva sugeriu e usar no controller o `params[:id]` acredito que funcionaria.

Reply:
boa @rpaffaro vou tentar assim